trial-config-service/src/test/resources/config.json

```diff
@@ -0,0 +1,15 @@
+{
+  "trial": [
+    {
+      "id": "test_1",
+      "trialName":"Hello World Trial",
+      "trialId":"prefix/1",
+      "trialSites":[
```

Comment:
here it says 'sites' where the parent is singular 'trial'

Reply:
Cant a Trial have many TrialSites? If so this is correct.

Reply:
I mean, if 'trial' is an array then it should be 'trials'
if it is singular, then 'trialSites' should be 'trialSite'.

both singular or plural

Reply:
file has been removed as a new testing approach has been agreed.